src/components/inputs/Select.test.tsx

```diff
@@ -0,0 +1,328 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { forEach, map, reject, slice } from 'lodash';
+
+import { render } from '../../test-utils';
+import { Select } from './Select';
+
+const items = [
+	{
+		label: 'hi',
+		value: '1'
+	},
+	{
+		label: 'hello',
+		value: '2'
+	},
+	{
+		label: 'good day',
+		value: '3'
+	},
+	{
+		label: 'goodnight',
+		value: '4'
+	},
+	{
+		label: 'nothing',
+		value: '5'
+	}
+];
+
+const extendedItems = [
+	...items,
+	{
+		label: 'unselectable',
+		value: '6',
+		disabled: true
+	}
+];
+
+describe('Select', () => {
+	const label = 'Select an item';
+	const onChange = jest.fn();
```

Comment:
I think it's better if this mock is initialized inside every single test, to avoid possible conflicts

Reply:
Yes, that is true, I will move them all inside every single test